libraries/freertos_plus/standard/tls/src/iot_tls.c

```diff
@@ -458,14 +458,18 @@ static int prvInitializeClientCredential( TLSContext_t * pxCtx )
                                                                     sizeof( configPKCS11_DEFAULT_USER_PIN ) - 1 );
     }
 
-    /* Get the handle of the device private key. */
-    xResult = xFindObjectWithLabelAndClass( pxCtx->xP11Session,
-                                            pkcs11configLABEL_DEVICE_PRIVATE_KEY_FOR_TLS,
-                                            CKO_PRIVATE_KEY,
-                                            &pxCtx->xP11PrivateKey );
+    if( CKR_OK == xResult )
```

Comment:
The above two if checks are ```if( 0 == xResult )```, would you please make them ```if( CKR_OK == xResult )```.  Since ```CKR_OK``` is defined to ```0```, it is just for consistency.

Reply:
yes good catch.  I reviewed the file to have a comparison to 0 for all PKCS returns.  (But I left the comparison to 0 for mbedtls returns).